bin/tools.py

```diff
@@ -1370,8 +1380,8 @@ def test(args):
     original_directory = Path().cwd()
     config.n_warn = 0
     config.n_error = 0
-    contest._contest_yaml = None
-    contest._problems_yaml = None
+    _contest_yaml = None
+    _problems_yaml = None
```

Comment:
CI error might be here? if this is a global variable now

Reply:
Most likely yes, because this only assigns `None` to the local variables in the `tools` module, rather than the (exported) variables in the `contest` module.